engine/src/test/java/org/destinationsol/testsupport/AssetsHelperInitializer.java

```diff
@@ -0,0 +1,57 @@
+package org.destinationsol.testsupport;
+
+import com.badlogic.gdx.Application;
+import com.badlogic.gdx.ApplicationAdapter;
+import com.badlogic.gdx.backends.headless.HeadlessApplication;
+import org.destinationsol.assets.AssetHelper;
+import org.destinationsol.assets.Assets;
+import org.destinationsol.modules.ModuleManager;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.parallel.ResourceLock;
+import org.terasology.gestalt.entitysystem.component.management.ComponentManager;
+
+/**
+ * Use it for tests, which used {@link org.destinationsol.assets.Assets}
+ */
+@ResourceLock("AssetsHelper")
+public interface AssetsHelperInitializer {
+    ThreadLocal<TestGameStateObject> state = new ThreadLocal<>();
+
+    @BeforeEach
+    default void initAssets() throws Exception {
+        TestGameStateObject stateObject = new TestGameStateObject();
+        state.set(stateObject);
+        stateObject.setApplication(new HeadlessApplication(new ApplicationAdapter() {
```

Comment:
This might be a personal style thing, but I find nested constructors like this hard to read. I think it'd be clearer if the `HeadlessApplication` was pulled out and given a descriptive variable name that indicates what it does. Something along the lines of:

```
Application mockApplication = new HeadlessApplication(new ApplicationAdapter() {
      ...
});
stateObject.setApplication(mockApplication);
```

I find that that makes it clearer what the object being passed in represents.